config/flags/flags.go

```diff
@@ -348,4 +348,14 @@ var (
 		Name:  "lachesis.suppress-frame-panic",
 		Usage: "Suppress frame missmatch error (when testing on historical imported/synced events)",
 	}
+
+	// StateDb
+	LiveDbCacheFlag = cli.Int64Flag{
+		Name:  "statedb.livecache",
+		Usage: "Size of live db cache in bytes.",
```

Comment:
Please document: 
- what is the default value?  This is needed, if not default, then a recommended value should be provided.
- does it accept 0 or -1 with special semantics? 

Users need to understand if they need to modify this values or if it is safe to ignore them.